mlflow/models/evaluation/base.py

```diff
@@ -155,6 +157,13 @@ def artifacts(self) -> Dict[str, "mlflow.models.EvaluationArtifact"]:
         """
         return self._artifacts
 
+    @property
+    def baseline_model_metrics(self) -> Dict[str, Any]:
```

Comment:
```suggestion
    def baseline_model_metrics(self) -> Optional[Dict[str, Any]]:
```

Reply:
Instead of making this nullable, can we make it an empty dict by default? See https://github.com/mlflow/mlflow/pull/6310/files#r933521860